src/irmin-pack/layered/layered_store.ml

```diff
@@ -333,8 +333,10 @@ end
 
 module Atomic_write
```

Comment:
This `Atomic_write` could be moved to his own file too. Maybe in a later PR

Reply:
good idea, I've added it to https://github.com/mirage/irmin/issues/1340